src/components/elements/List/style.scss

```diff
@@ -0,0 +1,80 @@
+:local {
+  @import '../../../styles/config';
+
+  .List {
+    margin-top: 0;
+    margin-bottom: 0;
+    padding-left: 0;
+    list-style: none;
+
+    > li {
```

Comment:
Usually, element selectors are expensive. Not sure in react components. We should think about more scalable CSS selector strategies such as BEM.